snippets/common/security.conf

```diff
@@ -5,6 +5,7 @@ add_header X-Content-Type-Options "nosniff" always;
 add_header Referrer-Policy "no-referrer-when-downgrade" always;
 #add_header Content-Security-Policy "default-src 'self' http: https: data: blob: 'unsafe-inline'; script-src 'self' 'unsafe-eval' 'unsafe-inline' https://www.googletagmanager.com https://www.google-analytics.com https://ssl.google-analytics.com https://www.googleadservices.com https://googleads.g.doubleclick.net https://www.google.com https://js.stripe.com https://cdnjs.cloudflare.com https://unpkg.com https://*.freecodecamp.org; object-src 'none';" always;
 add_header Content-Security-Policy "default-src 'self' http: https: data: blob: 'unsafe-inline' 'unsafe-eval';" always;
+add_header Content-Security-Policy "frame-ancestors 'self'";
```

Comment:
We should consolidate all this.

Reply:
Sorry about that, didn't realize you could put other directives on the same line after the semicolon.

Will fix this as soon as possible.